README.md

```diff
@@ -79,3 +79,22 @@ Run with coverage `python -m pytest --cov=src`
 Run with coverage and missing lines `python -m pytest --cov-report term-missing --cov=src`
 
 Run with coverage, missing lines and junit output `python -m pytest --cov-report term-missing --junitxml="result.xml" --cov=src`
+
+## Yara rules checks.
+
+There is a Jenkins job run on a schedule from the Jenkinsfile-check-rules file. This carries out the following steps. 
+* Builds the base yara image.
+* Builds the rules yara image.
+* Gets the most recent version from git.
+* Downloads the lambda zip file from S3.
+* Unzips the zip file and copies the exiting compiled rule file into the current directory.
+* Downloads the test files from tdr-antivirus-test-files-mgmt in S3.
+* Builds the docker image from the Dockerfile-run-tests file which copies the existing compiled rules and the test files to the container.
+* Runs this docker image. This runs a python script which runs these steps:
+    * Compares the rule identifiers in the old compiled rules file with the new one.
+    * If there are new rules in the new compiled rules, it will run the checks against the files. It will not detect if rules have been removed and it won't check if they've been changed either. I tried to do it based on hashes but it's likely to bring up too many false positives.
```

Comment:
Maybe reword this: 

> If there are new yara rules in the new compiled rules, it will run the antivirus checks against the test files from the S3 bucket. 
> * NOTE: It will not detect if rules have been removed, and it won't check if they've been changed either. I tried to do it based on hashes but it's likely to bring up too many false positives.

Reply:
Yeah that's better, I've changed it.